data/presets/leisure/pitch/tetherball.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "maki-pitch",
+    "fields": [
+        "{leisure/pitch}",
+        "access_simple"
+    ],
+    "geometry": [
+        "area",
+        "point"
```

Comment:
As noted on [the wiki](https://wiki.openstreetmap.org/wiki/Tag:sport%3Dtetherball#Alternative_tagging), `playground=tetherball` is also commonly used to indicate the tetherball pole itself. That would be a separate preset, but in the meantime, this preset would be available for points as well as areas, consistent with other kinds of pitches. I think that’s OK; it seems unlikely that a mapper unfamiliar with that tag would convert it into a tetherball court inside a tetherball court.